[2025-01-09T21:05:10.370+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: export_import_data.change_csv manual__2025-01-09T21:05:04.555973+00:00 [queued]>
[2025-01-09T21:05:10.380+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: export_import_data.change_csv manual__2025-01-09T21:05:04.555973+00:00 [queued]>
[2025-01-09T21:05:10.381+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-09T21:05:10.570+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): change_csv> on 2025-01-09 21:05:04.555973+00:00
[2025-01-09T21:05:10.577+0000] {standard_task_runner.py:60} INFO - Started process 3450 to run task
[2025-01-09T21:05:10.581+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'export_import_data', 'change_csv', 'manual__2025-01-09T21:05:04.555973+00:00', '--job-id', '2047', '--raw', '--subdir', 'DAGS_FOLDER/export_import_data.py', '--cfg-path', '/tmp/tmpilvm53f8']
[2025-01-09T21:05:10.587+0000] {standard_task_runner.py:88} INFO - Job 2047: Subtask change_csv
[2025-01-09T21:05:10.644+0000] {task_command.py:423} INFO - Running <TaskInstance: export_import_data.change_csv manual__2025-01-09T21:05:04.555973+00:00 [running]> on host 803f979fc22a
[2025-01-09T21:05:10.738+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='avbershits' AIRFLOW_CTX_DAG_ID='export_import_data' AIRFLOW_CTX_TASK_ID='change_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-01-09T21:05:04.555973+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-09T21:05:04.555973+00:00'
[2025-01-09T21:05:10.752+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-09T21:05:10.763+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=export_import_data, task_id=change_csv, execution_date=20250109T210504, start_date=20250109T210510, end_date=20250109T210510
[2025-01-09T21:05:10.802+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-09T21:05:10.833+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
